packages/components/src/input/src/Input.css

```diff
@@ -130,10 +121,15 @@
 }
 
 /* ICON | MEDIUM */
-.o-ui-input-has-icon input {
+.o-ui-input-has-icon {
     padding-left: var(--o-ui-input-padding-with-icon);
 }
 
+/* PASSWORD */
+.o-ui-password-input{
+    padding-right: 2.25rem;
```

Comment:
I don't think it's necessary since it's override by `.o-ui-input-has-button.o-ui-input`

![image](https://user-images.githubusercontent.com/794579/157533217-3bf1534c-0c3e-43d0-a77b-20f2d029aecf.png)